### PR TITLE
add: extra lines in index.tsx

### PR DESCRIPTION
**Pull request by opemipodisu**

### CODE REVIEW

File: store/index.tsx [hunk 1]
```diff
@@ -38,6 +38,7 @@ export const AppStateProvider: React.FC<{ children: React.ReactNode }> = ({ chil
     })
 
     useEffect(() => {
+        const hash = window.location.hash.slice(1);
```
Comment (tianzhou) [inline store/index.tsx:41]: why we need this

File: pages/index.tsx [hunk 2]
```diff
@@ -11,6 +11,8 @@ import Head from "next/head"
 
 interface IndexProps {}
 
+
```
Comment (tianzhou) [inline pages/index.tsx:14]: extra empty line

**Review by tianzhou — CHANGES_REQUESTED** [review]
Why we still remove robots.txt

File: store/index.tsx (hunk 1 above)
Comment (opemipodisu) [reply store/index.tsx:41]: that's a different thing entirely, i'm trying to fix the hash issue.

File: pages/index.tsx (hunk 2 above)
Comment (opemipodisu) [reply pages/index.tsx:14]: added the robots.txt there now.

Comment (opemipodisu) [reply pages/index.tsx:14]: i'll remove the extra lines, i was trying to do something there on state-management, but i got it wrong.

File: store/index.tsx (hunk 1 above)
Comment (tianzhou) [reply store/index.tsx:41]: Let's keep this out of this PR. You can use a different branch to do this

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Review by s-bytebase — APPROVED** [review]
LGTM

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**